decisions/log/0032-cross-sectoral-standards.md

```diff
@@ -61,8 +62,6 @@ We have also considered the possibility of disclosing a cross sectoral standard
         :: for version 2.1 of the PACT Framework
         : GHGP Product
         :: for the Greehouse Gas Protocol (GHGP) Product Standard
-        : PAS 2050
-        :: for the Publicly Available Specification (PAS) 2050
         : ISO14040-44
         :: for the ISO 14044-44 Standard
```

Comment:
Key states "ISO14040-44" and value states "ISO 14044-44"

Reply:
Great catch! Thank you! Will be fixed asap!

Reply:
Thanks for the catch! Now fixed (although note we are still awaiting final alignment from methodology colleagues on this list)